test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,70 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+        static private Platform envPlatform;
+
+        public static string GetNpmPrefixPath()
+        {
+            envPlatform = Platform.CurrentPlatform;
```

Comment:
where envPlatform is used here?

Reply:
https://github.com/appium/dotnet-client/blob/0a1df1b57f4c3d3ce0d40632d08779b75a51e001/test/integration/helpers/Npm.cs#L55-L59

Reply:
I would rather refactor that code because it creates an implicit contract/prependency between two methods., e.g. the one cannot be called unless the second has been invoked before.

Consider making envPlatform to a [lazy](https://stackoverflow.com/questions/4036812/c-sharp-lazy-loaded-automatic-properties) property instead

Reply:
@mykola-mokhnach, please review after my latest refactor and let me know if there is still a need for further refactor